ng-dev/pr/merge/merge-pull-request.ts

```diff
@@ -114,11 +121,13 @@ export async function mergePullRequest(prNumber: number, flags: PullRequestMerge
 async function createPullRequestMergeTool(flags: PullRequestMergeFlags) {
   try {
     const config = await getConfig();
+
     assertValidGithubConfig(config);
     assertValidPullRequestConfig(config);
+    assertValidCaretakerConfig(config);
```

Comment:
We can't perform this assertion on every pull request merge attempt as other repositories don't have a caretaker configuration.

We need to either perform the assertion in a non-breaking way, or perform the assertion later on.